odoo/addons/base/ir/ir_translation.py

```diff
@@ -544,8 +544,8 @@ def _check_value(self):
                 mname, fname = trans.name.split(',')
                 record = trans.env[mname].browse(trans.res_id)
                 field = record._fields[fname]
-                if callable(field.translate):
-                    src = trans.src
+                src = trans.src
+                if callable(field.translate) and not src.startswith('<img'):
```

Comment:
The element `img` is translatable, but that does not mean is always appear at the beginning of the source term. Consider:
```html
    <div>Please look at <img src="..."></div>
```
The source term to translate will be `'Please look at <img src="...">'`.

Reply:
I have checked all snippets and no where text is using like this. Everywhere if text is used, it is closed in <p> tag outside div of image tag. like -
               <p>Please look at </p>
               <div><img src="...."/><div>

Although, changed the code to find out the image tag. Now using regular expression for this.